### PR TITLE
Update duty cycle code

### PR DESCRIPTION
**Pull request by juehv**
Added suggestions from https://github.com/openaps/oref0/issues/1058 and removed emergency actions since a different script is rebooting when radio problems occur.

### CODE REVIEW

**Comment by juehv** [comment]
@scottleibrand can you please review the changes? It'll make the core much simpler and removes redundant functionality :)

File: bin/oref0-pump-loop.sh [hunk 1]
```diff
@@ -709,7 +669,7 @@ function get_settings {
         retry_return read_insulin_sensitivities 2>&3 >&4 || return 1
         retry_return read_carb_ratios 2>&3 >&4 || return 1
         retry_return openaps report invoke settings/insulin_sensitivities.json settings/bg_targets.json 2>&3 >&4 || return 1
-	#NON_X12_ITEMS=""
+    #NON_X12_ITEMS=""
```
Comment (scottleibrand) [inline bin/oref0-pump-loop.sh]: Tabs vs. spaces? Maybe just remove this old line.

Comment (juehv) [reply bin/oref0-pump-loop.sh]: This line isn't from me :D I just changed the duty cycle function ;)

Comment (scottleibrand) [reply bin/oref0-pump-loop.sh]: No, but the diff shows you changed the whitespace such that it is now indented correctly. I’m suggesting either fixing that or removing the line.

**Comment by juehv** [comment]
@scottleibrand other issues ?

**Pull request merged**